labman/db/process.py

```diff
@@ -3114,56 +3117,57 @@ def generate_prep_information(self):
-                LEFT JOIN qiita.sequencing_process_lanes USING (
+                LEFT JOIN labman.sequencing_process_lanes USING (
                     sequencing_process_id)
                 -- Retrieving pool information
-                LEFT JOIN qiita.pool_composition_components ON (
+                LEFT JOIN labman.pool_composition_components ON (
                     output_pool_composition_id = pool_composition_id)
                 -- Retrieving library prep information
-                LEFT JOIN qiita.library_prep_shotgun_composition ON (
+                LEFT JOIN labman.library_prep_shotgun_composition ON (
                     input_composition_id = composition_id)
-                LEFT JOIN qiita.primer_composition i5pc ON (
+                LEFT JOIN labman.primer_composition i5pc ON (
                     i5_primer_composition_id = i5pc.primer_composition_id)
-                LEFT JOIN qiita.primer_set_composition i5 ON (
+                LEFT JOIN labman.primer_set_composition i5 ON (
                     i5pc.primer_set_composition_id =
                     i5.primer_set_composition_id
                 )
-                LEFT JOIN qiita.primer_composition i7pc ON (
+                LEFT JOIN labman.primer_composition i7pc ON (
                     i7_primer_composition_id = i7pc.primer_composition_id)
-                LEFT JOIN qiita.primer_set_composition i7 ON (
+                LEFT JOIN labman.primer_set_composition i7 ON (
                     i7pc.primer_set_composition_id =
                     i7.primer_set_composition_id
                 )
                 -- Retrieving normalized gdna information
-                LEFT JOIN qiita.normalized_gdna_composition ngc USING (
+                LEFT JOIN labman.normalized_gdna_composition ngc USING (
                     normalized_gdna_composition_id)
-                LEFT JOIN qiita.composition c1 ON (
+                LEFT JOIN labman.composition c1 ON (
                     ngc.composition_id = c1.composition_id)
-                LEFT JOIN qiita.library_prep_shotgun_process lps ON (
+                LEFT JOIN labman.library_prep_shotgun_process lps ON (
                     lps.process_id = c1.upstream_process_id)
-                LEFT JOIN qiita.normalization_process np USING (
+                LEFT JOIN labman.normalization_process np USING (
                     normalization_process_id)
                 -- Retrieving compressed gdna information
-                LEFT JOIN qiita.compressed_gdna_composition cgc USING (
+                LEFT JOIN labman.compressed_gdna_composition cgc USING (
                     compressed_gdna_composition_id)
                 -- Retrieving gdna information
-                LEFT JOIN qiita.gdna_composition gc USING (gdna_composition_id)
-                LEFT JOIN qiita.composition c2 ON (
+                LEFT JOIN labman.gdna_composition gc
+                    USING (gdna_composition_id)
```

Comment:
I see a few places like this where what was one line previously has been broken into two.  I have no problem with it, just curious: what's the rationale?  Is it staying below the flake8-enforced line length, or is it some convention?

Reply:
Yup, flake8 line length convention.